Components/image.css

```diff
@@ -0,0 +1,8 @@
+/* Image */
+.img-res{
+    height: auto;
+    width: 100%;
+  }
+  .img-round{
+    border-radius: 50%;
+  }
```

Comment:
Just a suggestion, if rounded image can be given height-width, some what same, it would be a perfect circle.